app/views/divisions/index.html.haml

```diff
@@ -12,10 +12,9 @@
       %td
         = division.name
       %td{class: 'min-width', align: 'center'}
-        = link_to (content_tag(:span, nil, class: 'glyphicon glyphicon-trash')+' Delete'),
-                   division,
-                   method: :delete,
-                   data: { confirm: 'Are you sure?' },
-                   class: 'btn btn-xs btn-danger'
+        = form_tag division, method: :delete, data: { confirm: 'Are you sure?' } do
+          = button_tag(class: 'btn btn-xs btn-danger') do
```

Comment:
Oh yeah that's pretty gross. I managed to do this at one point but it was hard and I had to do a lot of googling. Whatever way works for right now I guess.